releases/release-1.31/release-notes/maps/pr-123905-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 123905
+releasenote:
+  text: |-
+    Services implemented a field selector for the ClusterIP and Type fields.
```

Comment:
```suggestion
    Services now have a field selector for the ClusterIP and Type fields. 
```

Reply:
Based on  
https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/release-notes#periodically-review-and-fix-new-release-notes
past tense should be used.
@npolshakova  please recommend

Reply:
Yep, one of the guidelines is to write like you're from the future (so use past tense in all release notes).